integrations/RedditPixel/browser.js

```diff
@@ -0,0 +1,85 @@
+import logger from "../../utils/logUtil";
+
+class RedditPixel {
+  constructor(config) {
+    this.advertiserId = config.advertiserId;
+    this.name = "REDDIT_PIXEL";
+  }
+
+  init() {
+    logger.debug("===In init RedditPixel===");
+
+    !(function (w, d) {
+      if (!w.rdt) {
+        var p = (w.rdt = function () {
+          p.sendEvent
+            ? p.sendEvent.apply(p, arguments)
+            : p.callQueue.push(arguments);
+        });
+        p.callQueue = [];
+        var t = d.createElement("script");
+        (t.src = "https://www.redditstatic.com/ads/pixel.js"), (t.async = !0);
+        var s = d.getElementsByTagName("script")[0];
+        s.parentNode.insertBefore(t, s);
+      }
+    })(window, document);
+
+    rdt("init", `${this.advertiserId}`);
```

Comment:
we can simply use this.advertiserId. No need to use string template

Reply:
updated with this.advertiserId without string template